### PR TITLE
Jets3t config

### PR DESCRIPTION
**Pull request by fparisotto**

### CODE REVIEW

File: templates/root/ephemeral-hdfs/conf/jets3t.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+s3service.internal-error-retry-max=50
+storage-service.internal-error-retry-max=50
+s3service.max-thread-count=8
+httpclient.max-connections=8
```
Comment (douglaz) [inline templates/root/ephemeral-hdfs/conf/jets3t.properties]: Why lower this value? The default is 20

Comment (fparisotto) [inline templates/root/ephemeral-hdfs/conf/jets3t.properties]: My mistake, will be fixed

File: templates/root/ephemeral-hdfs/conf/jets3t.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+s3service.internal-error-retry-max=50
```
Comment (douglaz) [inline templates/root/ephemeral-hdfs/conf/jets3t.properties]: I would use a lower value, something between 10-15 minutes of total await time. 35 seems good:

```
>>> sum(50 * i**2 /1000./60. for i in range(1, 35+1))
12.425
>>> [50 * i**2 /1000./60. for i in range(1, 35+1)][-1]
1.0208333333333333
```

**Comment by douglaz** [comment]
LGTM.

**Pull request merged**